docs/versionIndex.md

```diff
@@ -83,156 +80,36 @@ of components organized by ADF libraries.
 
 <!--v200 start-->
 
--   [Accordion group component](core/accordion-group.component.md)
```

Comment:
Is it an error or has been moved somewhere else?

Reply:
It looks like loads of components are missing from the file. This is a new one - I'll investigate it and update the PR when I've found what is wrong.

Reply:
Ah, no, it's OK - I've found what is wrong. I've only done the update for the content services docs and so the version index only contains those files. I'll update it and commit it again.